### PR TITLE
add controller for receiving box req form payload

### PR DESCRIPTION
**Pull request by h-m-m**
### Description

This is part of the work on #3. This takes the data we expect from @bolducp's box request form in #94 and starts persisting it to the database.

### Type of change

* New feature (non-breaking change which adds functionality)
* This change requires a documentation update

### Testing

We still have not done the end-to-end test yet. Since this provides a concrete endpoint for the React side, hopefully we'll be able to test end-to-end once this is merged in.

### CODE REVIEW

File: app/controllers/box_request_triage_controller.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    #TODO some of the existing fields (is_interested_in_counseling_services, is_interested_in_health_services, question_re_affect, question_re_current_situation, question_re_if_not_self_completed, question_re_referral_source) are not in our db for the time being
+    [:first_name,
+     :last_name,
+     :email,
+     :street_address,
+     :city,
+     :state,
+     :zip,
+     :ok_to_email,
```
Comment (maebeale) [inline app/controllers/box_request_triage_controller.rb:20]: we had originally anticipated the following additional fields on Requester:
- phone
- county
- underage

Comment (maebeale) [reply app/controllers/box_request_triage_controller.rb:20]: were these removed from the form (they do seem to be present in the db)?

Comment (maebeale) [reply app/controllers/box_request_triage_controller.rb:20]: for underage, that can become a tag on the BoxRequest, but think we should still add it to the Requester table (even tho that data will become stale, i think we would want to retain that they were underage when we added them to the system)

**Review by maebeale — APPROVED** [review]

**Review by abachman — APPROVED** [review]
📏 rules

**Pull request merged**